jkube-kit/doc/src/main/asciidoc/inc/enricher/autotls/_jkube_openshift_autotls.adoc

```diff
@@ -3,39 +3,20 @@
 ==== jkube-openshift-autotls
 
 Enricher which adds appropriate annotations and volumes to enable OpenShift's automatic
-https://docs.openshift.org/latest/dev_guide/secrets.html#service-serving-certificate-secrets[Service Serving Certificate Secrets].
+https://docs.openshift.com/container-platform/4.1/authentication/certificates/service-serving-certificate.html[Service Serving Certificate Secrets].
```

Comment:
nit: Can we use the latest URL https://docs.openshift.com/container-platform/4.10/security/certificates/service-serving-certificate.html